### ISSUE TITLE
feature request: alt text via MSC3552

### ISSUE THREAD

**Issue by lavenderdotpet** [commetchat/commet#1031]
https://github.com/matrix-org/matrix-spec-proposals/pull/3552

any chance of ui for alt text being support via MSC3552

description taken from the spec proposal
``
Optional - An m.alt_text block to represent alternative/descriptive text for the image. This is used as an accessibility feature, and per the block's definition above should only contain a plain text representation at the moment. Clients are encouraged to assume there is no alt text if no plain text representations are present. For clarity, this value would be supplied to the alt attribute of an img node in HTML.
}
``
